cmd/kops/get_federation.go

```diff
@@ -82,19 +84,21 @@ func RunGetFederations(context Factory, out io.Writer, options *GetFederationOpt
 			return strings.Join(f.Spec.Members, ",")
 		})
 		return t.Render(federations, out, "NAME", "CONTROLLERS", "MEMBERS")
-	} else if output == OutputYaml {
+
+	case OutputYaml:
```

Comment:
I agree that OutputTable should be separate.  At least it isn't obvious that it should be the same, so that could always be a separate PR.  Sorry not to be clear!